datastore/simple/transformers/payload.go

```diff
@@ -0,0 +1,81 @@
+package transformers
+
+import (
+	"time"
+
+	"github.com/teslamotors/fleet-telemetry/protos"
+)
+
+func PayloadToMap(payload *protos.Payload, includeTypes bool) map[string]interface{} {
+	convertedPayload := make(map[string]interface{}, len(payload.Data)+2)
+	convertedPayload["Vin"] = payload.Vin
+	convertedPayload["CreatedAt"] = payload.CreatedAt.AsTime().Format(time.RFC3339)
```

Comment:
nit: should we log array length of Data?

Reply:
Opting not to